### ISSUE TITLE
spyder 3.2 startup

### ISSUE THREAD

**Issue by sumant1811** [spyder-ide/spyder#11157]
getting this message on spyder 3.2 startup.

    plugin = mod.PLUGIN_CLASS(self)
AttributeError: '_ModuleMock' object has no attribute 'PLUGIN_CLASS'

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.7.4
* Qt Versions: 5.12.6, PyQt5 5.12.3 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  2.1.1 (OK)
pycodestyle >=2.3:  2.5.0 (OK)
pygments >=2.0   :  2.4.2 (OK)
pandas >=0.13.1  :  0.25.3 (OK)
numpy >=1.7      :  1.17.4 (OK)
sphinx >=0.6.6   :  2.2.2 (OK)
rope >=0.9.4     :  0.14.0 (OK)
jedi >=0.9.0     :  0.14.1 (OK)
nbconvert >=4.0  :  5.6.0 (OK)
sympy >=0.7.3    :  None (NOK)
cython >=0.21    :  None (NOK)
qtconsole >=4.2.0:  4.6.0 (OK)
IPython >=4.0    :  7.8.0 (OK)
pylint >=0.25    :  2.4.2 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest Spyder 3 (3.3.6) version. Please update.

**Comment by sumant1811** [comment]
Thanks. Problem Solved.
